packages/jest-dom-mocks/src/document.ts

```diff
@@ -0,0 +1,42 @@
+export interface MockWindow extends Window {
+  addEventListener: jest.Mock<{}> & typeof window.addEventListener;
+  removeEventListener: jest.Mock<{}> & typeof window.removeEventListener;
+}
+
+export default class Document {
```

Comment:
It's weird to call this `Document` but internally it's doing all it's mocking on `window`